api/app/models.py

```diff
@@ -26,3 +26,22 @@ def validate_highlights(cls, v, values):
         if v:
             assert len(v) == len(values['paragraphs'])
         return v
+
+class SearchQueryResponse(BaseModel):
+    query_id: str
+    response: List[Article]
+
+class SearchLogData(BaseModel):
+    query_id: str
+    result_id: str
+    position: int
+
+class SearchLogType(str, Enum):
+    query = 'query'
+    collapsed = 'collapsed'
+    expanded = 'expanded'
+    clicked = 'clicked'
+
+class SearchVertical(str, Enum):
```

Comment:
Sorry if this is a dummy question, but is vertical a commonly used name? Maybe "SearchCorpus"?

Reply:
It's a pretty common term at some companies, I used it at LinkedIn a lot

Reply:
For example this post: https://engineering.linkedin.com/search/unifying-linkedin-search-experience

Reply:
Cool, thanks! I didn't know about this term